tests/src/common/Wsk.scala

```diff
@@ -75,13 +77,20 @@ case class WskProps(
     def overrides = Seq("--apihost", apihost, "--apiversion", apiversion)
 }
 
-class Wsk extends RunWskCmd {
+class Wsk(val usePythonCLI: Boolean) extends RunWskCmd {
```

Comment:
you can do the same without the `def this()` with a default argument on the constructor:
`class Wsk(usePythonCLI: Boolean = true)`